install/Installer.sh

```diff
@@ -328,67 +269,38 @@ spinnaker:
       bucket: $CONFIG_BUCKET
 EOL
 
-  echo $HAL_USER > /opt/spinnaker/config/halyard-user
-
-  cat > $halconfig_dir/uninstall.sh <<EOL
+  cat > /tmp/uninstall-halyard.sh <<EOL
 #!/usr/bin/env bash
 
 if [[ \`/usr/bin/id -u\` -ne 0 ]]; then
-  echo "$0 must be executed with root permissions; exiting"
+  echo "uninstall-halyard.sh  must be executed with root permissions; exiting"
   exit 1
 fi
 
-read -p "This script uninstalls Halyard and deletes all of its artifacts, are you sure you want to continue? (Y/n): " yes
+read -p "This script uninstalls Halyard and deletes all of its artifacts, are you sure you want to continue? (y/N): " yes
 
 if [ "\$yes" != "y" ] && [ "\$yes" != "Y" ]; then
   echo "Aborted"
   exit 0
 fi
 
-apt-get purge spinnaker-halyard
+apt-get purge -y spinnaker-halyard
 
 echo "Deleting halconfig and artifacts"
 rm /opt/spinnaker/config/halyard* -rf
 rm $halconfig_dir -rf
```

Comment:
I don't see anything that defines `$halconfig_dir` anymore.

Reply:
Apologies, I was  fast asleep when I tried to put it back 🤦 .  Its fixed now, thank you.